### ISSUE TITLE
Italicized title in tufte_handout

### ISSUE THREAD

**Issue by ernietedeschi** [rstudio/tufte#87]
Just a minor question, not even necessarily a feature request: I'd like to change the formatting of my title typeface to be a normal font, not italicized, using tufte_handout/PDF output. But I'd like to keep the section headers italicized as-is. How/where would I do this? Is there a tufte_handout option already implemented? Or would I need to directly edit a template file somewhere?

---

By filing an issue to this repo, I promise that

- [x] I have fully read the issue guide at https://yihui.name/issue/.
- [x] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('tufte')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('rstudio/tufte')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [x] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by cderv** [comment]
Hi, 

There is no option in the package to do specifically this. Unfortunately, I don't know exactly how you should do it. If you want to customize a PDF output style, you need to do that in Latex directly. (For HTML, it would be with CSS).

Please follow the issue guide regarding questions: https://yihui.org/issue/#got-a-question
You got better place to ask the community for help with this kind of question. There is also special TeX community that can help with that. 

I'll close this as this is not an issue, and we track issues and FR request only in GH

**Comment by ernietedeschi** [comment]
No, I'd respectfully ask you keep it open, as there is bound to be a better answer than the one you gave. This is an issue with the tufte package specifically, not TeX or R, which is why I asked it here. I'm not asking how to unset italics in TeX, I'm asking which file in the tufte package I ought to edit. Surely the people who created it and maintain it must know that.

**Comment by cderv** [comment]
> This is an issue with the tufte package specifically, not TeX or R, which is why I asked it here.  

How would you do that it Tex without the this package ? 

You seem to describe it as an issue, but there is nothing to reproduce or your did not share what you tried and what works in Tex or R and not with this **tufte** package that would lead to believe there is an issue to solve here. 

FWIW, the Pandoc template used by this package is here: 
https://github.com/rstudio/tufte/blob/master/inst/rmarkdown/templates/tufte_handout/resources/tufte-handout.tex

You can add a latex content inside the template as with any other PDF document with rmarkdown 
https://bookdown.org/yihui/rmarkdown/pdf-document.html#includes-1

I believe the `tufte_handout` format is using the documentclass `tufte-handout` which provided by the latex `tufte` package as said in the [book](https://bookdown.org/yihui/rmarkdown/tufte-handouts.html#fn5) : https://tufte-latex.github.io/tufte-latex/
If you want to customize the style provided by this tool, I believe you need to deal with Latex for that. 

That is why I closed the issue because we track bug report and feature request here, and not specific questions, as the issue guide is saying. 

Happy to reopen for you to give more feedback if there is indeed an issue with the package when you tried to use something that would work with Latex directly. You did not share that, so I may have misunderstood in the first place.

**Comment by cderv** [comment]
It has been 5 years without reply so I am closing.
